infrastructure/stacks/cloudwatch-dashboard/main.tf

```diff
@@ -0,0 +1,141 @@
+resource "aws_cloudwatch_dashboard" "cloudwatch_dashboard" {
+
+  dashboard_name = var.cloudwatch_performance_dashboard_name
+  dashboard_body = <<EOF
+{
+    "widgets": [
+        {
+            "height": 6,
+            "width": 23,
+            "y": 0,
+            "x": 0,
+            "type": "metric",
+            "properties": {
+                "view": "timeSeries",
+                "stacked": false,
+                "metrics": [
+                    [ "ENV", "${var.environment}" ]
+                ],
+                "region": "${var.aws_region}"
+            }
+        },
+        {
+            "height": 6,
+            "width": 6,
+            "y": 16,
+            "x": 6,
+            "type": "metric",
+            "properties": {
+                "view": "timeSeries",
+                "stacked": false,
+                "metrics": [
+                    [ "ENV", "${var.environment}" ]
+                ],
+                "region": "${var.aws_region}"
+            }
+        },
+        {
+            "height": 6,
+            "width": 6,
+            "y": 10,
+            "x": 0,
+            "type": "metric",
+            "properties": {
+                "metrics": [
+                    [ "AWS/Lambda", "ConcurrentExecutions", "FunctionName", "${var.project_id}-${var.environment}-event-processor" ],
+                    [ ".", "Errors", ".", ".", { "stat": "Sum" } ],
+                    [ ".", "Invocations", ".", "." ],
+                    [ ".", "Duration", ".", "." ],
+                    [ ".", "Throttles", ".", ".", { "stat": "Sum" } ],
+                    [ ".", "Duration", ".", ".", { "stat": "Minimum" } ],
+                    [ "...", { "stat": "Maximum" } ],
+                    [ "...", { "stat": "TM(10%:90%)" } ]
+                ],
+                "view": "timeSeries",
+                "stacked": false,
+                "region": "${var.aws_region}",
+                "title": "Event Processor",
+                "period": 300,
+                "stat": "Average"
+            }
+        },
+        {
+            "height": 4,
+            "width": 23,
+            "y": 6,
+            "x": 0,
+            "type": "metric",
+            "properties": {
+                "view": "timeSeries",
+                "stacked": false,
+                "metrics": [
+                    [ "AWS/SQS", "NumberOfMessagesSent", "QueueName", "${var.project_id}-${var.environment}-fifo-queue.fifo" ],
```

Comment:
instead of rebuilding the queue name can we not use the existing makefile variable TF_VAR_fifo_queue_name that way if the name changes then this will still be correct, possibly same for lambda function above?

Reply:
Ya, it makes sense. I will update accordingly